src/Layout/Navigation/RecentItemWorkspaceActions.tsx

```diff
@@ -102,12 +102,15 @@ class NavigationItemWorkspaceActions extends React.PureComponent<Props, State> {
         createAction(WorkspaceAction.STOP_WORKSPACE),
       );
     }
-    // todo should be add with https://github.com/eclipse/che/issues/19514
-    // if (status !== WorkspaceStatus.STOPPING && status !== DevWorkspaceStatus.TERMINATING) {
-    //   dropdownItems.push(
-    //     createAction(WorkspaceAction.RESTART_WORKSPACE),
-    //   );
-    // }
+    if (
+      status !== WorkspaceStatus.STOPPING &&
+      status !== WorkspaceStatus.STOPPED &&
+      status !== DevWorkspaceStatus.TERMINATING
+    ) {
+      dropdownItems.push(
+        createAction(WorkspaceAction.RESTART_WORKSPACE),
```

Comment:
Stopped devworkspace seems to have restart action shown
![Screenshot_20210624_134019](https://user-images.githubusercontent.com/5887312/123249607-cf983e00-d4f1-11eb-879d-43efea57366d.png)

Reply:
I have fixed it